README.md

```diff
@@ -53,11 +53,17 @@ See our docs for more info on how to use the library.
 
 ### Examples
 
-To run an example, clone the repo and run the following commands
-
+To run an example, clone the repo and follow these steps
+1. Copy contents of `/apps/web/.env.example` into a new file, `/apps/web/.env`
+2. Copy contents of `/examples/.env.example` into a new file, `/examples/.env.example` and add your [OpenAI API Key](https://openai.com/api/) to the `OPENAI_API_KEY` field
```

Comment:
what about `apps/demos/gpt-prisma-seed/.env.example`?

Reply:
i'll put that in the readme in the `gpt-prisma-seed` directory since you only need to make the change if you're running that example